starlark/value.go

```diff
@@ -1275,7 +1294,10 @@ func (s *Set) IsSubset(other Iterator) (bool, error) {
 	}
 }
 
+// Warning: the resulting set has the iteration order of other.
```

Comment:
Rather than add these Warnings and TODOs, we should fix the underlying problem in this CL.

Reply:
Done.